site/content/blogs/release-v128.en.md

```diff
@@ -33,10 +33,46 @@ Copilot v1.28 brings several new features and improvements:
     See the section [Overview](../docs/concepts/overview.en.md) for a more detailed introduction to AWS Copilot.
 
 ## Static Site service type
+Copilot's newest workload type, Static Site, provisions and configures everything you need to create a static website hosted by Amazon S3 and fronted by an Amazon CloudFront distribution.
 
 ### Static Site Upload Experience
+As with other services, begin your Static Site creation with the [`copilot init`](../docs/commands/init.en.md) or [`copilot svc init`](../docs/commands/svc-init.en.md) command. You may use the `--sources` flag to pass in the path(s) (relative to your project root) to your static asset directories and/or files. Alternatively, you may select the directories/files when prompted.
 
-### Integrate With CloudFront
+A manifest will be populated and stored in the `copilot/[service name]` folder. There, you may adjust your asset specifications if you'd like. By default, all directories will be uploaded recursively. If that's not what you want, leverage the `exclude` and `reinclude` fields to add filters. The available pattern symbols:  
+
+`*`: Matches everything  
+`?`: Matches any single character  
+`[sequence]`: Matches any character in sequence  
+`[!sequence]`: Matches any character not in sequence  
+
+```yaml
+# The manifest for the "example" service.
+# Read the full specification for the "Static Site" type at:
+#  https://aws.github.io/copilot-cli/docs/manifest/static-site/
+
+# Your service name will be used in naming your resources like S3 buckets, etc.
+name: example
+type: Static Site
+files:
+  - source: src/someDirectory
+    recursive: true
+  - source: someFile.html
+
+# You can override any of the values defined above by environment.
+# environments:
+#   test:
+#     files:
+#       - source: './blob'
+#         recursive: true
+#         destination: 'assets'
+#         exclude: '*'
+#         reinclude:
+#           - '*.txt'
+#           - '*.png'
+```
+For more on `exclude` and `reinclude` filters, go [here](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/s3/index.html#use-of-exclude-and-include-filters).
+
+The [`copilot deploy`](../docs/commands/deploy.en.md) or [`copilot svc deploy`](../docs/commands/svc-deploy.en.md) command will provision and launch your static website: creating an S3 bucket and uploading your chosen local files to that bucket, adding necessary permissions, generating a CloudFront distribution with the S3 bucket as the origin, etc. Under the hood, your Static Site service will have a CloudFormation stack, just like other Copilot workloads.
```

Comment:
nit: for "necessary permissions" maybe the bucket objects are only accessible publicly through cloudfront.

> Under the hood, your Static Site service will have a CloudFormation stack, just like other Copilot workloads.

Hmm this reminds me of a concern that if users know what they would expect from this blog post. It sounds like readers need to know Copilot and AWS pretty well to understand why they would want to use static site in the first place.

Maybe for the blog post we put a more concrete use case (for example i want to create a single-page application): from why we want to use Static site (can't i do the same thing with existing Copilot?) to how can I solve the problem with Static site. Show them steps they need to take and some screenshots (ok to be pending since the impl is not done yet) for what they would expect after the deployment (like files are indeed correctly selected and uploaded to the s3, alias used for CF works fine)